style.scss

```diff
@@ -140,13 +140,14 @@ img {
   z-index: 9999;
   color: var(--gray1);
   background-color: var(--gray5);
-  box-shadow: 0 4px 6px #80808033; // gray3 but with 20% alpha
+  box-shadow: 4px 4px 6px #80808033; // gray3 but with 20% alpha
   border-radius: 15px;
 
   @include mobile {
     right: 0;
     bottom: 0;
-    border-radius: 0 4px 0 #80808033;
+    box-shadow: 0 -4px 6px #80808033 
```

Comment:
-4 Y ? Isn't it 4 ?

Reply:
On mobile the banner is stuck to the bottom of the screen

Reply:
OK then no shadow on mobile